### PR TITLE
Fetch

### PR DESCRIPTION
**Pull request by Andy19936**

### CODE REVIEW

File: src/pages/DevIndex.js [hunk 1]
```diff
@@ -5,22 +5,30 @@ import { Card, CardTitle, Col, CardImg } from "reactstrap";
...
-          id="indexImg"
-        />
         <Col sm="6">
           {this.props.jobs &&
             this.props.jobs.map((job) => {
               return (
-                <Card className="indexCards" body key={job.id}>
+                <Card
+                  className="indexCards"
+                  style={{
```
Comment (WalkerTexas-coder) [inline src/pages/DevIndex.js:17]: Is there a reason that these styles are here and not in the CSS?  I notice the indexCards className. Not looking to replicate this cards style?

**Review by WalkerTexas-coder — COMMENTED** [review]
Left a question about styling location

File: src/pages/DevIndex.js (hunk 1 above)
Comment (Andy19936) [reply src/pages/DevIndex.js:17]: yeah the styling isnt permanent we were using this to figure out how to style react components for some reason we couldnt get the css file to handle it for us so we are going to use inline styling instead

Comment (Andy19936) [reply src/pages/DevIndex.js:17]: we are finalizing the style after lunch

Comment (WalkerTexas-coder) [reply src/pages/DevIndex.js:17]: Cool I'll approve this then

**Review by WalkerTexas-coder — APPROVED** [review]
Looks good, can't wait to see it!

**Pull request merged**